cmd/main.go

```diff
@@ -19,115 +19,28 @@ package main
 import (
 	"log"
 
-	"github.com/spf13/cobra"
-
-	"sigs.k8s.io/kubebuilder/cmd/internal"
 	"sigs.k8s.io/kubebuilder/cmd/version"
-	"sigs.k8s.io/kubebuilder/internal/config"
-	"sigs.k8s.io/kubebuilder/pkg/scaffold"
+	"sigs.k8s.io/kubebuilder/pkg/cli"
+	pluginv1 "sigs.k8s.io/kubebuilder/pkg/plugin/v1"
```

Comment:
We should not care about V1 since it is deprecated and should no longer be used.
Also, IHMO we need to remove the V1 code in order to kept the thing simple. 
However, since it is already done in the PR we can merge with.

Reply:
Removing v1 may be easier after merging the feature branch with master. @DirectXMan12 @mengqiy thoughts on this? Regardless, out of scope for this PR.

Reply:
Just to track here: We have a PR for that already:  https://github.com/kubernetes-sigs/kubebuilder/pull/1355